jdaviz/configs/imviz/helper.py

```diff
@@ -442,6 +447,25 @@ def link_image_data(app, link_type='pixels', wcs_fallback_scheme='pixels', wcs_u
                                  f" Clear markers with viewer.reset_markers() first")
 
     refdata, iref = get_reference_image_data(app)
+
+    # if linking via WCS, add WCS-only reference data layer:
+    insert_base_wcs_layer = (
+        link_type == 'wcs' and
+        base_wcs_layer_label not in [d.label for d in app.data_collection] and
```

Comment:
I think this is probably fine for now, but could cause problems if/when we add support for _renaming_ data layers.  Would it make sense instead to just check if there are _any_ of these orientation/WCS-only layers already existing and only add the default one if not?  Otherwise, maybe just a comment here will do the trick (I'm also adding a note to the ticket to rename data entries that points back here [:cat:](https://jira.stsci.edu/browse/JDAT-1979))

Reply:
I agree that renaming could cause problems here. I would just advocate for not allowing renaming of the base layer.